Apigee/Mint/Developer.php

```diff
@@ -418,43 +418,25 @@ public function topUpPrepaidBalance($new_balance, $developer_or_company_id = nul
 
     public function getRevenueReport($report)
     {
-        $url = '/mint/organizations/'
-            . rawurlencode($this->config->orgName)
-            . '/developers/'
-            . rawurlencode($this->email)
-            . '/revenue-reports';
+        $url = rawurlencode($this->email) . '/revenue-reports';
         $content_type = 'application/json; charset=utf-8';
         $accept_type = 'application/octet-stream; charset=utf-8';
 
-        $this->setBaseUrl($url);
-        $this->post(null, $report, $content_type, $accept_type);
-        $this->restoreBaseUrl();
+        $this->post($url, $report, $content_type, $accept_type);
         $response = $this->responseText;
         return $response;
     }
 
     public function saveReportDefinition($report_def)
     {
-        $url = '/mint/organizations/'
-            . rawurlencode($this->config->orgName)
-            . '/developers/'
-            . rawurlencode($this->email)
-            . '/report-definitions';
-        $this->setBaseUrl($url);
-        $this->post(null, $report_def);
-        $this->restoreBaseUrl();
+        $url = rawurlencode($this->email) . '/report-definitions';
+        $this->post($url, $report_def);
     }
 
     public function getReportDefinitions()
     {
-        $url = '/mint/organizations/'
-            . rawurlencode($this->config->orgName)
-            . '/developers/'
-            . rawurlencode($this->email)
-            . '/report-definitions';
-        $this->setBaseUrl($url);
-        $this->get();
-        $this->restoreBaseUrl();
+        $url = rawurlencode($this->email) . '/report-definitions';
```

Comment:
Did you mean to change the value of `$url` here? If so, an explanation would help.

Reply:
Same as above:
```
$base_url = '/mint/organizations/' . rawurlencode($config->orgName) . '/developers';
```
`$url` appended to `$base_url`, that's why we do not need to change `$base_url`